core/dbt/contracts/graph/model_config.py

```diff
@@ -376,15 +374,6 @@ def finalize_and_validate(self: T) -> T:
         self.validate(dct)
         return self.from_dict(dct)
 
-    def replace(self, **kwargs):
```

Comment:
This was removed because it did extra munging of the keys using the field_mapping, which is no longer needed. It's not currently used anywhere. If somebody wanted equivalent functionality they could use the "replace" function. I'm not a fan of the "Replaceable" class; I think it adds confusion and bloat and actually did a branch with a trial removal, but decided it wasn't worth it at that time. Since this wasn't used, I don't think the behavior with defaults mattered :)  But I suspect it was accidental.